src/trading/postSwapOrder.test.ts

```diff
@@ -0,0 +1,144 @@
+import { getQuoteWithSigner } from './getQuote'
+
+jest.mock('cross-fetch', () => {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const fetchMock = require('jest-fetch-mock')
+  // Require the original module to not be mocked...
+  const originalFetch = jest.requireActual('cross-fetch')
+  return {
+    __esModule: true,
+    ...originalFetch,
+    default: fetchMock,
+  }
+})
```

Comment:
Is this soimething we want to mock always or just in this file?

Reply:
The mock is specific for sell/buy orders, so I don't think we can use it by default